### ISSUE TITLE
[Bug] [a11y]Error Page-Information is being repeated from "Address Not Found" section

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#8635]
Pre-conditions
- Have TalkBack enabled

## Steps to reproduce
1. Open app.
2. Tap on the search bar.
3. Insert an invalid address.
4. With talkback, activated tap on the 2 checkpoints phrases:
* Check the address for typing errors such as ww.example.com instead of www.example.com.
* If you are unable to load any pages, check your devices' data or Wi-FI connection.

### Expected behavior
The information is not repeated.
### Actual behavior
The following information is repeated 2 times:
* Check the address for typing errors such as ww.example.com instead of www.example.com.
* If you are unable to load any pages, check your devices' data or Wi-FI connection.

### Device information

* Android device: Google Pixel C (Android 8.0.0)
* Fenix version: Firefox Preview Beta 4.0.0-beta.1

**Comment by asadotzler** [comment]
I can reproduce this. The list items are being read twice. This is an accessibility P2. The content is available but is spoken twice which is an annoyance and prevents Fenix from being a delightful experience for people with disabilities.

**Comment by mcarare** [comment]
This was previously fixed in https://github.com/mozilla-mobile/android-components/pull/5299. But changes have been made to error pages since. I am going to take a look at this.

**Comment by mcarare** [comment]
@asadotzler I can fix this, but I have a follow-up question:

The text inside the first item is using a `<strong>` tag to make the difference between "**ww**" and "**www**" clearer. This is a semantic markup that adds meaning for screen readers, so that text would be read separately from the rest of the text.
In the current implementation, the implicit `listitem` role of that `<li>` item determines Talkback to read all the text at once. But after I remove that role, so that it would be read only once, not as part of a list, the text would be read separately, split by the word marked with `<strong>` tag.

What I am asking is if I should allow the split. IMO is the correct screen reader behavior, but just wanted to confirm that with you. Thank you!

**Comment by mcarare** [comment]
Fix has landed via AC, QA please check. TY!

**Comment by sflorean** [comment]
Verified as fixed on Nightly 7/24 with LG G7 FIT (Android 8), following the steps from the description.

However, going to cnn.com and after the ETP shield is displayed, the content "Every time the shield is purple, Firefox Nightly has blocked trackers on a site. Tap for more info." is read **twice**.

@mcarare should I file a new issue, or should we consider this one?

**Comment by mcarare** [comment]
@sflorean Please file a new issue, as it is related to a completely different component. TY!
